### PR TITLE
Adds styling to the FTL settings tab, fixes #166

### LINKED ISSUE #166: The FTL settings tab is unstyled
The tab's contents are plaintext. It should be styled like the network tab.

### PR DESCRIPTION
**Pull request by isdampe**
Styles the contents of the FTL tab to match the network tab as described in #166.

Result:
Screen Shot 2019-06-15 at 8 42 00 AM

Signed-off-by: Richard Denton <isdampe@gmail.com>

### CODE REVIEW

File: src/components/settings/FTLInfo.tsx [hunk 1]
```diff
@@ -24,13 +25,47 @@ class FTLInfo extends Component<FTLInfoProps & WithTranslation, {}> {
...
+        <FormGroup row>
+          <Label className="bold" for="sqliteVersion" sm={4}>
+            {t("SQLite version")}
+          </Label>
+          <Col sm={8}>
+            <Input
+              plaintext
+              readOnly
+              id="sqliteversion"
+              value={this.props.sqliteVersion.toLocaleString()}
```
Comment (Mcat12) [inline src/components/settings/FTLInfo.tsx]: The SQLite version should be shown as it is given to us, as it is not a number, date, or other such data that can be formatted based on locale.

I see this was carried on from the previous code, so no worries!

File: src/components/settings/FTLInfo.tsx [hunk 2]
```diff
@@ -24,13 +25,47 @@ class FTLInfo extends Component<FTLInfoProps & WithTranslation, {}> {
...
+        <FormGroup row>
+          <Label className="bold" for="ipv4_address" sm={4}>
+            {t("Queries")}
+          </Label>
+          <Col sm={8}>
+            <Input
+              plaintext
+              readOnly
+              id="queries"
+              value={this.props.queries}
```
Comment (Mcat12) [inline src/components/settings/FTLInfo.tsx]: This should use `.toLocaleString()`.

I see this was carried on from the previous code, so no worries!

File: src/components/settings/FTLInfo.tsx [hunk 3]
```diff
@@ -24,13 +25,47 @@ class FTLInfo extends Component<FTLInfoProps & WithTranslation, {}> {
...
+        </FormGroup>
+        <FormGroup row>
+          <Label className="bold" for="sqliteVersion" sm={4}>
+            {t("SQLite version")}
+          </Label>
+          <Col sm={8}>
+            <Input
+              plaintext
+              readOnly
+              id="sqliteversion"
```
Comment (Mcat12) [inline src/components/settings/FTLInfo.tsx]: The label refers to `sqliteVersion`, but this ID is `sqliteversion` (lowercase V).

File: src/components/settings/FTLInfo.tsx [hunk 4]
```diff
@@ -24,13 +25,47 @@ class FTLInfo extends Component<FTLInfoProps & WithTranslation, {}> {
...
+        <FormGroup row>
+          <Label className="bold" for="filesize" sm={4}>
+            {t("Filesize")}
+          </Label>
+          <Col sm={8}>
+            <Input
+              plaintext
+              readOnly
+              id="filesize"
+              value={this.props.fileSize.toLocaleString()}
```
Comment (Mcat12) [inline src/components/settings/FTLInfo.tsx]: This should specify that the value is in bytes. The original code did this by adding the `B` unit prefix to the end.

**Comment by Mcat12** [comment]
The CI is failing due to a formatting issue in the file. Try `npm run format`.

**Comment by isdampe** [comment]
Thanks for the feedback. Amendments to follow

**Comment by isdampe** [comment]
Fixed in 1d560ac3ccaa48aba9adb4c7bbe51e291c86db4a

**Pull request merged**